intelligence-services/latest/using/index.md

```diff
@@ -45,10 +45,10 @@ Follow these steps to use the default (i.e. out-of-the-box) configuration of Int
         For example, you can add any of the default renditions:
 
         ```bash
-        aiFeatures, aiLabels, aiTextract, aiSpeechToText, webvtt, aiPiiEntities 
+        aiFeatures, aiLabels, aiTextract
         ```
 
-        > **Note:** If you leave the text field empty, all three of the default renditions will be requested. The default renditions are `aiFeatures` for Amazon Comprehend, `aiLabels` for Amazon Rekognition of images, and `aiTextract` for Amazon Textract.
+        > **Note:** If you leave the text field empty all of the default renditions will be requested. The default renditions are, **aiLabels** -> AI Labels, **aiTextract** -> AI Text Lines, aiFeatures -> AI Creative Works, AI Dates, AI Events, AI Organizations, AI People, AI Places, AI Products, AI Quantities, AI Things, **aiPiiEntities** -> AI PII Entities, **aiSpeechToText** -> AI Transcript, **webVTT** -> n/a.  
```

Comment:
This doesn't look right ;-) 

The aspect mapping should be in another bullet point plus an example screenshot as well.

Reply:
Done. I havent added a screen shot.